LayoutTests/fast/dom/collection-children-backwards.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<body>
+<div id="container">
+	<div id="child0"></div>
+	<div id="child1"></div>
+	<div id="child2"></div>
+</div>
+<script src="../../resources/js-test.js"></script>
+<script>
+
+description('This tests iterating element.children backwarwds. WebKit should return the last element when indexed for the last item.')
```

Comment:
Maybe don't say "WebKit" in case this becomes a WPT (which, ideally, it would be).

Reply:
ok.